devices/terncy.js

```diff
@@ -38,7 +38,7 @@ module.exports = [
         ota: ota.zigbeeOTA,
         meta: {battery: {dontDividePercentage: true}},
         exposes: [e.battery(), e.action(['single', 'double', 'triple', 'quadruple', 'rotate']),
-            exposes.text('direction', ea.STATE)],
+            exposes.text('direction', ea.STATE), exposes.numeric('number', ea.STATE).withDescription('Number of turns')],
```

Comment:
Shouldn't `number` be `action_number`?

Reply:
No, if it should, also instead `direction` there should be `action_direction`. Also tested it.
![image](https://user-images.githubusercontent.com/12080749/132118601-2bd62496-bd1f-499f-bfe3-97171e4df03b.png)

Reply:
Or am I unaware of some convention?

Reply:
Ah sorry, you still have the legacy integration enabled, try disabling it: https://www.zigbee2mqtt.io/devices/TERNCY-SD01.html#device-type-specific-configuration and make sure to remove your `data/state.json` to purge the old attributes.

Reply:
Stopped service, set `legacy: false` for this device in `devices.yaml`, removed `state.json` and that seemed to break my scripts for this device, did not investigate much (maybe missing states), why was that needed?